### DISCUSSION TITLE
make image without contribution from specific grid region

### DISCUSSION THREAD

**Discussion by seanandrews** [dullemond/radmc3d-2.0#40] [General]
Hi,

I'd like to try and make a (spectral line) image, using a spherical polar coordinate grid, that specifically avoids the mirror symmetry about the midplane.  For example, I'd like to exclude emission from negative (cylindrical) z values.  When I make the grid, (R, Theta) are positive values.  Is there a simple way to turn off mirror symmetry in Theta?  (I'm happy to do this by setting densities or abundances to 0 there too, but am not sure how that would work specifically.)

thanks,
Sean

**Comment by dullemond** [comment]
Hi Sean,
RADMC-3D automatically switches to mirroring if it detects that the larges theta value is pi/2 to high precision. You can avoid this by setting the max theta value to be slightly less than pi/2. You might run onto the warning
```            write(stdo,*) 'SAFETY STOP: It seems that you want to use mirror symmetry in the'
            write(stdo,*) '   equatorial plane by putting the highest theta_i to pi/2. But'
            write(stdo,*) '   presumably you did not use high enough precision in the '
            write(stdo,*) '   theta_i list in the amr_grid.inp file. The mirror symmetry'
            write(stdo,*) '   is not set. Stopping for safety. If you are absolutely sure'
            write(stdo,*) '   you know what you are doing, you can comment out this stop.'
            stop
```
In that case remove this warning and the stop by commenting them out.
Let me know if this solves this issue for you. Given that this is presumably only useful for "testing experiments", not for realistic models, I never bothered to implement a "soft" way to do this ;-).
Cheers,
Kees

**Reply by seanandrews** [reply]
Hi Kees,

Awesome....this worked perfectly.  And indeed, this is for "testing experiments" (i.e., my brain is incapable of dealing with projecting 3-D structures into a 2-D plane).  

thanks!
Sean
